lib/tasks/gather.js

```diff
@@ -702,8 +704,8 @@ class TaskGather extends SttTask {
     this.logger.debug(`Starting timoutTimer of ${this.timeout}ms`);
     this._clearTimer();
     this._timeoutTimer = setTimeout(() => {
+      // If continuousASR in use then extend by the asr window for more transcripts.
       if (this.isContinuousAsr) this._startAsrTimer();
-      if (this.interDigitTimer) return; // let the inter-digit timer complete
```

Comment:
this seems questionable....why would we resolve here if there is an active interdigit timer